templates/taxi/car_detail.html

```diff
@@ -1,22 +1,30 @@
 {% extends "base.html" %}
 
 {% block content %}
+  <h1>{{ car.model }}</h1>
+  <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
   <h1>
-    {{ car.model }}
-    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
-      Delete
-    </a>
-
-    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
-      Update
-    </a>
+    Drivers
+    {% if user in car.drivers.all %}
+      <a href="{% url 'taxi:car-remove-driver' pk=car.pk %}" class="btn btn-danger link-to-page">
+        Remove me from this car
+      </a>
+    {% else %}
+      <a href="{% url 'taxi:car-assign-driver' pk=car.pk %}" class="btn btn-success link-to-page">
+        Assign me to this car
+      </a>
+    {% endif %}
   </h1>
-  <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
   <hr>
   <ul>
     {% for driver in car.drivers.all %}
```

Comment:
you use here many to many relationship but did you fetched it from db? because otherwise here will be N+1 issue